client/src/Components/Note/Note.js

```diff
@@ -1,25 +1,124 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 
 import './Note.css';
 
 class Note extends Component {
-  state = {
-    note: ''
+  constructor(props) {
+    super(props);
+    this.state = {
+      editnote: '',
+      viewnote: '',
+      messagebox: '',
+      messageboxcolor: '',
+      editview: 'false',
+      user: this.props.user
+    };
+  }
+
+  //this happens when something changes on the form
+  onChangeNote = event => {
+    this.setState({ editnote: event.target.value });
+  };
+
+  //when submit button is clicked
+  submitChecker = event => {
+    event.preventDefault();
+    if (this.state.editnote !== '') {
+      this.onSubmitSuccess(event);
+    } else {
+      this.onSubmitFailure();
+    }
+  };
+
+  messageResetTimer = () => {
+    setTimeout(() => {
+      this.setState({ messagebox: '' });
+    }, 3500);
+  };
+
+  //when submit checker says no
+  onSubmitFailure = () => {
+    this.setState({ messageboxcolor: 'red' });
+    this.setState({
+      messagebox: 'Please fill out your note before you submit!'
+    });
+    this.messageResetTimer();
+  };
+
+  //when submit checker says yes
+  onSubmitSuccess = async () => {
+    const noteSubmission = {
+      text: this.state.editnote,
+      user: this.state.user
+    };
+    try {
+      let response = await axios.post('http://localhost:8000', noteSubmission);
```

Comment:
You'll probably want to change this to reference the backend url variable from the .env. So something like 
```
let response = await axios.post(`${process.env.BACKEND_URL}/note`, noteSubmission);
```
(name of the endpoint might be different)

Reply:
Good idea, resolved.